pkg/Makefile.am

```diff
@@ -25,9 +25,12 @@ dist_sshmanifest_DATA = pkg/ssh/manifest.json
 # one built file in dist/ which we use as dependency
 DIST_STAMP = $(srcdir)/dist/static/manifest.json
 
+# https://blog.jgc.org/2011/07/gnu-make-recursive-wildcard-function.html
+rwildcard=$(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
+
 # dynamic pkg → dist dependency, to rebuild the webpack if any web related file changes
 # exclude automake unit test log files
-PKG_INPUTS = $(shell find $(srcdir)/pkg -type f ! -name 'test*.trs' ! -name 'test*.log')
+PKG_INPUTS = $(filter-out %.log %.trs,$(call rwildcard,$(srcdir)/pkg,*))
```

Comment:
Why do we get test log files in pkg ?

Reply:
It's from pkg/users/test-list-public-keys.sh. Unfortunately automake puts the corresponding *.log and *.trs next to the test itself (into the source dir).